lib/cisco_node_utils/bgp_af.rb

```diff
@@ -448,6 +448,79 @@ def default_dampening_suppress_time
     end
 
     #
+    # Distance (Getter/Setter/Default)
+    #
+    def distance_set(ebgp, ibgp, local)
+      if ebgp == default_distance_ebgp && ibgp == default_distance_ibgp &&
+         local == default_distance_local
+        set_args_keys(state: 'no', ebgp: distance_ebgp, ibgp: distance_ibgp,
+                      local: distance_local)
+      else
```

Comment:
I don't think we need this if-else check for defaults. The ebgp/ibgp/local vars are set to integer values so if the outcome is the same (doing a 'no' vs. setting the default values) then just set the values regardless; ie.

```
    def distance_set(ebgp, ibgp, local)
      set_args_keys(state: , ebgp: ebgp, ibgp: ibgp, local: local)
      config_set('bgp_af', 'distance', @set_args)
    end
```